lib/finch/response.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Finch.Response do
+  alias __MODULE__
+
+  defstruct [
+    :status,
+    headers: [],
+    data: ""
```

Comment:
Should we go with `body` instead of `data`?

Reply:
Good point. `data` makes sense within Mint, but at this level of abstraction `body` probably makes more sense